Dockerfile

```diff
@@ -14,6 +26,17 @@ RUN ln -s /usr/lib/x86_64-linux-gnu/libhwloc.so.1 /usr/lib/x86_64-linux-gnu/libh
 RUN wget http://ftp.us.debian.org/debian/pool/main/i/icu/libicu63_63.2-3_amd64.deb
 RUN dpkg -i libicu63_63.2-3_amd64.deb
 
+# Install Rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
+RUN echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> $HOME/.zshrc
+RUN echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> $HOME/.bashrc
+RUN . $HOME/.cargo/env
+RUN rustup default nightly
```

Comment:
do we need nightly?

Reply:
Unfortunately yes, `no_std` + `alloc` needs nightly

Reply:
See https://github.com/rust-lang/rust/issues/51540

The `alloc_error_handler` attribute has not been stabilized yet, unlike `global_allocator` and `panic_handler`...